ndmg/graph/graph.py

```diff
@@ -156,15 +162,7 @@ def save_graph(self, graphname, fmt='edgelist'):
                     - Output graph format
         """
         self.g.graph['ecount'] = nx.number_of_edges(self.g)
-        g = nx.convert_node_labels_to_integers(self.g, first_label=1)
```

Comment:
Are we no longer supporting any other output formats? Also, why are we not having nodes as integers anymore? Some graph software including igraph in R gets mad when they're not, in my experience.

Reply:
Where below? I don't see any comment addressing this

Reply:
In particular regards to the output graph format

Reply:
Oh; we de-supported everything but csvs because they were the fastest/most efficient; can add back the others though if you'd prefer; particularly, graphmls would be of use probably

Reply:
i don't need to support the older formats.  @gkiar do you want us to for some reason?
they have been quite difficult for many of our collaborators

Reply:
I think that weighted edge-lists are the best for diffusion by a long shot.

Since we still have a `format` flag from the command-line, and have it specified in various function definitions (sometimes with different defaults), this wasn't *actually* de-supported at every level so this intention was very unclear.

I also don't see the reason to not support writing simple formats like graphml, as it was a conditional and a 1 line statement for writing... The better solution is to just change the default to whatever format you want, but continue to support the relevant writers that NetworkX gives us for free.

Reply:
the background on graphml is as follows: graphml was basically a fail, it requires a specific version of networkx because networkx is not backwards compatible, which means it also requires a specific version of python.  we could update to the most recent version of graphml, but then we'll have to keep doing that presumably.  
so, it is much easier for us to not support it at all, or specify that we only support a now basically useless version.  

so, i vote weighted edge-lists only.
if people really want it in graphml format, they can put it there.

Reply:
I believe that is actually the case for gpickle, since the base pickle library changed between Python 2 and 3, but not graphml since it's an XML format. I think we removed (or should remove) gpickle support for that reason, but haven't seen graphml cause the same problem.

Either way, the command-line parser needs to be updated to reflect what options we are actually supporting.